stories/webapp/hierarchy/HierarchyCollection.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { number, bool } from 'prop-types';
```

Comment:
We should have a prize for anyone who can write a script to run over our whole codebase to refactor all the PropTypes in this way 🏆